fmriprep/workflows/bold/confounds.py

```diff
@@ -74,14 +74,20 @@ def init_bold_confs_wf(mem_gb, metadata, name="bold_confs_wf"):
             BIDS metadata for BOLD file
 
     **Inputs**
-
-        bold
+        bold_orig
```

Comment:
I added this here to ensure that non_steady_state correction did not use a denoised bold to calculate the number of non-steady state volumes. I haven't actually tested it to see if it would make a difference, but I wanted to be on the same side.